app/sentinel/background/initialization/run.go

```diff
@@ -58,6 +58,9 @@ import (
 func RunInitCommands(ctx context.Context) {
 	cid := ctx.Value("correlationId").(*string)
 
+	waitInterval := env.InitCommandRunnerWaitIntervalForSentinel()
```

Comment:
This is an escape hatch for us to provide as much wait time as needed in case things don’t reconcile for one reason or another.

When a bunch of other issues are fixed, we won’t need this, but still it’s good to have the option.